pkg/kubectl/cmd/create_configmap.go

```diff
@@ -49,7 +49,13 @@ var (
 		  kubectl create configmap my-config --from-file=key1=/path/to/bar/file1.txt --from-file=key2=/path/to/bar/file2.txt
 
 		  # Create a new configmap named my-config with key1=config1 and key2=config2
-		  kubectl create configmap my-config --from-literal=key1=config1 --from-literal=key2=config2`)
+		  kubectl create configmap my-config --from-literal=key1=config1 --from-literal=key2=config2
+
+		  # Create a new configmap named my-config from the key=value pairs in the file
+		  kubectl create configmap my-config --from-file=path/to/bar
+			
+			# Create a new configmap named my-config from an env file
+			kubectl create configmap my-config --from-env-file=path/to/bar`)
```

Comment:
I'd suggest using `.env` in the examples, like `... --from-env-file=path/to/file.env`.